Markdown/cmm-2.0.14/release-notes/2-restrictions.md

```diff
@@ -47,7 +47,6 @@ project in OpenStack Horizon and then try to access Kibana, the Kibana dashboard
 be not loading properly.
 
 
-### Workaround:
-
-For all browser versions suppported: Please use the default options of your browser to reload the page.
+* Workaround:
+  For all browser versions suppported: Please use the default options of your browser to reload the page.
```

Comment:
Some typo here - 'suppported' should be  'supported'.

Reply:
Done